test/Jamfile.v2

```diff
@@ -91,5 +92,6 @@ project
         [ run cpp_load_test.cpp   : : cpp_plugin ]
         [ run cpp_import_test.cpp   : : cpp_plugin ]
         [ run template_method_linux_test.cpp : : cpp_plugin ]
+        [ run gcc_ctti_type_name_parser_test.cpp : : cpp_mangle_plugin ]
```

Comment:
Please rename the file, don't make it GCC specific. The test would probably work well on clang and icc if it works on GCC.